app/assets/stylesheets/admin.scss

```diff
@@ -119,6 +119,10 @@ which may be more easily removed once we no longer wish admin layouts to have a
     display: flex;
     align-items: center;
     justify-content: space-between;
+    padding: 0.75rem 1.25rem;
```

Comment:
Bootstrap classes does contains `card-header` too and therefore these 4-lines of code has been added directly from that bootstrap class so that when we remove bootstrap it does not create any problem.